appserver/extras/payara-micro/payara-micro-core/src/main/java/fish/payara/micro/impl/PayaraMicroImpl.java

```diff
@@ -2373,6 +2404,14 @@ private void packageUberJar() {
             props.setProperty("payaramicro.userLogFile", userLogFile);
         }
 
+        if (accessLogInterval != Integer.MIN_VALUE) {
+            props.setProperty("payaramicro.accessLogInterval", Integer.toString(accessLogInterval));
+        }
+        
+        if (accessLogPrefix != null) {
+            props.setProperty("payaramicro.customlog", accessLogPrefix);
```

Comment:
Could do with a better name

Reply:
```suggestion
            props.setProperty("payaramicro.accessLogPrefix", accessLogPrefix);
```

Reply:
Actually is this even used anywhere? Doesn't seem to be...?

Reply:
I've spotted that after making the PR, and I added it for consistency with the rest of the properties